2025-08-22 14:54:21 - back_end.server - ERROR - Chat error: 'RiskBot' object has no attribute '_req_variable'
Traceback (most recent call last):
  File "C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\back_end\server.py", line 138, in analyze_contract
    analysis_result = risk_bot.contract_check(file_path)
  File "C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\utils\LLM_Analyzer.py", line 398, in contract_check
    response = get_strict_talking(self,"bot/AnalyzeContract",message)
  File "C:\Users\z0054unn\Documents\Siemens-GitLab\Third-party\third-party-clearance\utils\tools.py", line 593, in get_strict_talking
    response = model._req_variable(promptName, variable)
               ^^^^^^^^^^^^^^^^^^^
AttributeError: 'RiskBot' object has no attribute '_req_variable'. Did you mean: '_req_varibale'?
